src/main/java/se/sundsvall/webmessagecollector/service/scheduler/MessageCacheScheduler.java

```diff
@@ -101,6 +114,7 @@ private void fetchAttachments(MessageEntity message) {
 			} catch (Exception e) {
 				LOG.error(String.format("Unable to fetch attachment(s) for message with id '%s'", message.getId()), e);
```

Comment:
Not in this PR, but shouldn't this make use of parameterized logging instead?:
e.g:
`LOG.error("Unable to fetch attachment(s) for message with id '{}'", message.getId(), e);`

Reply:
Fixed. Anything on the code in this PR?